examples/data_preprocessing/[WIP]_chexpert_dataset/Prepare_CheXpert_data.ipynb

```diff
@@ -0,0 +1,958 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# paths to training images\n",
+    "image_paths_train = [os.path.join(path[: path.find(\"CheXpert-v1.0-small\")], \"CheXpert-v1.0-small\", p) for p in training_set[\"Path\"]]\n",
+    "\n",
+    "# paths to validation images\n",
+    "image_paths_valid = [os.path.join(path[: path.find(\"CheXpert-v1.0-small\")], \"CheXpert-v1.0-small\", p) for p in validation_set[\"Path\"]]\n",
+    "\n",
+    "sample_image = Image.open(image_paths_train[0]).convert('RGB')\n",
+    "plt.imshow(sample_image)\n",
+    "plt.show()\n",
+    "print(\"Dimensions of image:\", sample_image.size)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "All images in the (downsampled) dataset have a dimensionality of around 390 x 320 pixels, however, the individual image sizes vary. (Garbin et al. (2021))\n",
+    "\n",
+    "In the following, a transform sequence is defined, which will be used to preprocess the images. The images are first resized to 224 x 224 pixels, since this is a requirement for many pre-trained CNNs in PyTorch, and then normalized with the mean and standard deviation from ImageNet. Such transformations have previously been applied by several other submissions regarding CheXpert on GitHub such as:\n",
+    "[this](https://github.com/gaetandi/cheXpert/blob/master/cheXpert_final.ipynb) and [this](https://github.com/Stomper10/CheXpert/blob/master/CheXpert_DenseNet121_FL.ipynb)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "transform_list = transforms.Compose([\n",
+    "    transforms.Resize((224, 224)),\n",
+    "    transforms.ToTensor(),\n",
+    "    transforms.Normalize([0.485, 0.456, 0.406], [0.229, 0.224, 0.225]) # normalization from ImageNet\n",
+    "    ])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The following class will be used to transform the images and prepare the dataset for PyTorch's DataLoader. Moreover, the uncertainty labels will be handled according to either the \"U-Ones\", \"U-Zeroes\", \"U-Ignore\" or \"U-MultiClass\" approach as they were described in the origianl CheXpert paper by Irvin et al. (2019).\n",
+    "\n",
+    "You can specify how the uncertainty labels for each individual pathology will be handled by providing a list of the names of the pathologies to the arguments to_ones, to_zeros and to_ignore. to_ignore changes the uncertainty label (-1.0) to nan. Not specifying any pathology in any of the arguments to_ones, to_zeros or to_ignore will train the model with the \"U-MultiClass\" approach. \n",
```

Comment:
ah it is related to the previous methods, right? I see it now :D but I still would prefer a somewhat more consistent explanation. Maybe with a bullet list (four options, brief explanation + corresponding argument)?